data/core/units/dunefolk/Harrier.cfg

```diff
@@ -22,7 +22,7 @@ units/dunefolk/skirmisher/#enddef
     [abilities]
         {ABILITY_SKIRMISHER}
     [/abilities]
-    description= _ "While harriers can fulfill a similar role to striders in hampering enemy forces, their epithet is actually derived from the harrier hawks they employ to maintain contact with the bulk of a Dunefolk troop. Most commonly found operating behind enemy lines, harriers are usually tasked with providing useful intelligence on enemy troop movements, which alone is enough of an edge for most Dunefolk armies to outmaneuver their foes. However, when the time is right, these agile warriors gather together and strike at the rear of weakened supply lines or flanks. A rain of bolas followed by a piercing thrust of spears is often enough to severely cripple the target, allowing the main Dunefolk force to close in and finish the job."
+    description= _ "While Harriers can fulfill a similar role to striders in hampering enemy forces, their epithet is actually derived from the harrier hawks they employ to maintain contact with the bulk of a Dunefolk troop. Most commonly found operating behind enemy lines, harriers are usually tasked with providing useful intelligence on enemy troop movements, which alone is enough of an edge for most Dunefolk armies to outmaneuver their foes. However, when the time is right, these agile warriors gather together and strike at the rear of weakened supply lines or flanks. A rain of bolas followed by a piercing thrust of spears is often enough to severely cripple the target, allowing the main Dunefolk force to close in and finish the job."
```

Comment:
Reference to Strider, since it's a unit name, should be capitalised as well, shouldn't it?

There's a second instance of harriers that's not capitalised as well.